documentation/source/library_reference.rst

```diff
@@ -66,6 +66,15 @@ classes used within ``cocotb``.
     :members:
     :member-order: bysource
 
+The Scheduler
+-------------
+
+.. currentmodule:: cocotb.scheduler
+
+.. autoclass:: Scheduler
```

Comment:
Maybe:
```
.. note::
    The scheduler object should generally not be interacted with directly - the only part of it that you'll need is encapsulated in :func:`cocotb.fork`, everything else works behind the scenes.

Reply:
Also perhaps add:
```
.. py:variable:: scheduler
    The global scheduler instance
```
Worth looking up what the correct Sphinx syntax is for that.

Reply:
Added this with a docstring on ``scheduler`` and ``.. autodata:: cocotb.scheduler``.